mleap-core/src/main/scala/ml/combust/mleap/core/clustering/LDA.scala

```diff
@@ -0,0 +1,398 @@
+package ml.combust.mleap.core.clustering
```

Comment:
This mere copy of Spark code base. Yet to be tested since the MLeap pipeline loading fails at LDA.